include/kitty/bit_operations.hpp

```diff
@@ -313,7 +314,14 @@ int64_t find_last_one_bit( const TT& tt )
 template<typename TT>
 int64_t find_first_bit_difference( const TT& first, const TT& second )
 {
-  assert( first.num_vars() == second.num_vars() );
+  if constexpr ( std::is_same<TT, partial_truth_table>::value )
```

Comment:
@msoeken  Can I use `if constexpr`? It generates `warning: constexpr if is a C++17 extension`

Reply:
Hmm I made all of them 
```C++
#ifdef constexpr
if constexpr ( std::is_same<TT, partial_truth_table>::value )
#else
if ( std::is_same<TT, partial_truth_table>::value )
#endif
```
Ugly but doesn't generate warnings.

Reply:
You can update the standard in https://github.com/msoeken/kitty/blob/4f90ce79e8e489f18ce89adb090e4c07fd7c76d1/CMakeLists.txt#L4